Dockerfile.prod

```diff
@@ -0,0 +1,60 @@
+# BUILDER
```

Comment:
Why do you have that file here:
1. called Dockerfile.prod?
2. Mostly commented


In general if you want to set up the docker setup differently based on an environment, ideally you either want to set different docker files or deal with them based on the global variable of the environment. 

In general you want to avoid having multiple files like this because it makes it extremely difficult to maintain.